src/vivarium_census_prl_synth_pop/tools/cli.py

```diff
@@ -263,3 +266,33 @@ def jobmon_make_results_runner(
         seed,
         artifact_path,
     )
+
+
+@click.command()
+@click.argument("processed_results_dir", type=click.Path(exists=True))
+@click.option(
+    "--pdb",
+    "with_debugger",
+    is_flag=True,
+    help="Drop into python debugger if an error occurs.",
+)
+@click.option("-v", "verbose", count=True, help="Configure logging verbosity.")
+@click.option(
+    "-l",
+    "--state",
+    type=str,
+    default="RI",
+    show_default=True,
+    help="State to subset process results to obtain a smaller dataset for one specific geographic location.",
```

Comment:
Nit: mention in the help text that we expect the two letter postal abbreviation.